locales/en-US/messages.properties

```diff
@@ -99,10 +99,10 @@ faq_intro=Below are the most frequently asked questions about donating to the Mo
 faq_item_1_paragraph=<a href="https://donate.mozilla.org/">Visit this link to make a secure online donation right now via credit card or PayPal</a>. For additional ways to give -- like check, bitcoin, or bank transfer -- visit our <a href="https://donate.mozilla.org/ways-to-give">Ways to Give page</a>.
 faq_item_2_header=How does Mozilla ensure my personal information is protected?
-faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/en-US/privacy/">Mozilla Privacy Policy</a>.
+faq_item_2_paragraph_a=We care about the privacy of our donors and take our <a href="https://www.mozilla.org/privacy/principles/">Privacy Principles</a> seriously. Mozilla believes individuals should be in control of their data, and maintains multi-layered security controls and practices. To learn more, read the <a href="https://www.mozilla.org/privacy/">Mozilla Privacy Policy</a>.
 faq_item_2_paragraph_b=In certain instances, when you donate to Mozilla via a third party like <a href="https://www.paypal.com/webapps/mpp/ua/privacy-full">PayPal</a> or <a href="https://stripe.com/privacy/">Stripe</a>, those organizations also collect user data.
 faq_item_3_header=Why do you need my address in order to process a donation?
-faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/en-US/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="donate.mozilla.org/ways-to-give">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
+faq_item_3_paragraph=We understand that your privacy is very important. We ask for a minimum amount of information required to process credit card donations, including billing addresses. This allows our payment processor to verify your identity, process your payment, and prevent fraudulent charges to your credit card. We keep your information private -- if you have questions, please refer to our <a href="https://www.mozilla.org/privacy/principles/">Privacy Policy</a>. If you would rather not fill in your information on our online donation form, you can <a href="donate.mozilla.org/ways-to-give">mail a check</a> or donate directly <a href="https://sendto.mozilla.org/page/contribute/Give-Now">via Paypal</a>.
```

Comment:
<a href="donate.mozilla.org/ways-to-give"> should be <a href="https://donate.mozilla.org/ways-to-give">